frontend/packages/operator-lifecycle-manager/src/types.ts

```diff
@@ -246,3 +246,7 @@ export type OperatorGroupKind = {
     lastUpdated: string;
   };
 } & K8sResourceKind;
+
+export type Annotation = {
```

Comment:
I'd use `ObjectMetadata['annotations']` or export the type here:

https://github.com/openshift/console/blob/master/frontend/public/module/k8s/index.ts#L39